00:00:00.000874 main     VirtualBox COM Server 6.1.16 r140961 win.amd64 (Oct 16 2020 16:33:17) release log
00:00:00.000877 main     Log opened 2020-11-23T15:51:02.678573100Z
00:00:00.000878 main     Build Type: release
00:00:00.000881 main     OS Product: Windows 10
00:00:00.000881 main     OS Release: 10.0.18363
00:00:00.000882 main     OS Service Pack: 
00:00:00.129057 main     DMI Product Name: 20L6S9U200
00:00:00.135862 main     DMI Product Version: ThinkPad T480
00:00:00.135878 main     Firmware type: BIOS
00:00:00.135884 main     Host RAM: 32426MB (31.6GB) total, 25938MB (25.3GB) available
00:00:00.135888 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.135888 main     Process ID: 3244
00:00:00.135889 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.200121          VirtualBox: object creation starts
00:00:00.200898          Home directory: 'F:\PortableApps\VirtualBox\data\.VirtualBox'
00:00:00.200989          Installed Drivers:
00:00:00.202803            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.16.40961)
00:00:00.203336            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.16.40961)
00:00:00.203611            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.16.40961)
00:00:00.204325            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.16.40961)
00:00:00.210214          Loading settings file "F:\PortableApps\VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.211784          usbLibGetDevices: Starting USB device enumeration
00:00:00.213205          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.213247          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 7
00:00:00.213285          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.213319          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 9
00:00:00.214321          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 15
00:00:00.214421          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 1
00:00:00.215662          usbLibGetDevices: Found 10 USB devices, 0 captured
00:00:00.215731          USB: Unknown USB device detected (idVendor: 0x06cb, idProduct: 0x009a)
00:00:00.215743          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x0316)
00:00:00.215757          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56a6)
00:00:00.215777          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0a2b)
00:00:00.215806          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0af1)
00:00:00.293135          HostDnsMonitor: initializing
00:00:00.300941          HostDnsMonitor: updating information
00:00:00.300962          HostDnsMonitor: old information
00:00:00.300963            no server entries
00:00:00.300963            no domain set
00:00:00.300964            no search string entries
00:00:00.300964          HostDnsMonitor: new information
00:00:00.300965            server 1: 127.0.0.1
00:00:00.300965            domain: ni.corp.natinst.com
00:00:00.300966            no search string entries
00:00:00.404785          VD: VDInit finished with VINF_SUCCESS
00:00:00.407253          Loading settings file "F:\PortableApps\VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:00:00.410923          Loading settings file "F:\PortableApps\VirtualBox\data\.VirtualBox\Machines\Debug\Debug.vbox" with version "1.14-windows"
00:00:00.420137          OCI: Local config file 'F:\PortableApps\VirtualBox\data\.VirtualBox\oci_config' does not exist
00:00:00.420237          OCI: Original config file 'C:\Users\inthirua\.oci\config' does not exist
00:00:00.420238          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.420244          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.420264          VirtualBox: object created
00:00:21.416406          Saving settings file "F:\PortableApps\VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:21.555070          Saving settings file "F:\PortableApps\VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:27.158212 main     VirtualBox: object deletion starts
00:00:27.401218 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\PortableApps\VirtualBox\data\.VirtualBox\Machines\LinuxLive\HardDisks/LinuxLive.vmdk' cannot be closed because it is still attached to 2 virtual machines}, preserve=false aResultDetail=0
00:00:27.407134 main     HostDnsMonitor: shutting down ...
00:00:27.412136 main     HostDnsMonitor: shut down
00:00:27.421479 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:00:27.427540 main     VirtualBox: object deleted
